platforms/software/dependency-management/src/main/java/org/gradle/api/internal/catalog/LibrariesSourceGenerator.java

```diff
@@ -664,18 +671,17 @@ private void writeDeprecationLog(boolean deprecated) throws IOException {
     }
 
     private void writePlugin(String alias, PluginModel plugin, boolean asProvider) throws IOException {
-        indent(() -> {
-            writeLn("/**");
-            writeLn(" * Creates a plugin provider for " + alias + " to the plugin id '" + plugin.getId() + "'");
-            writeVersionInformation(plugin.getVersionRef(), plugin.getVersion());
-            String context = plugin.getContext();
-            if (context != null) {
-                writeLn(" * This plugin was declared in " + sanitizeUnicodeEscapes(context));
-            }
-            writeLn(" */");
-            String methodName = asProvider ? "asProvider" : "get" + toJavaName(leafNodeForAlias(alias));
-            writeLn("public Provider<PluginDependency> " + methodName + "() { return createPlugin(\"" + alias + "\"); }");
-        });
+        writeLn("/**");
+        writeLn(" * Plugin provider for <b>" + alias + "</b> with plugin id <b>" + plugin.getId() + "</b> and");
+        writeVersionInformation(plugin.getVersionRef(), plugin.getVersion());
+        String context = plugin.getContext();
+        if (context != null) {
+            writeLn(" * <p>");
```

Comment:
No, closing tag is not required for `<p>`. I guess it's a special case that javadoc format supports, but I always use it, because the non-rendered javadoc a bit cleaner.

FYI, https://stackoverflow.com/a/33787857/8144243

Reply:
got it! thanks @alllex